plugins/contiv/pod.go

```diff
@@ -241,6 +245,7 @@ func (s *remoteCNIserver) podTAP(request *cni.CNIRequest, podIPNet *net.IPNet) *
 	return &linux_intf.LinuxInterfaces_Interface{
 		Name:    "pod-" + s.tapTmpHostNameFromRequest(request),
 		Type:    linux_intf.LinuxInterfaces_AUTO_TAP,
+		Mtu:     s.config.MTUSize,
```

Comment:
A question - where is the MTU set for the interface to the Linux stack?

Reply:
Yes, check host.go changes